lib/App/DuckPAN.pm

```diff
@@ -79,6 +79,13 @@ has duckpan_packages => (
 	builder => 1,
 );
 
+has fathead_output => (
```

Comment:
@moollaza Yes - unfortunately the command would have to be in that form.

May I suggest an alternative in the form of a Role then?

If you have (something like):

```perl
# lib/App/DuckPAN/Option/FatHead.pm
package App::DuckPAN::Option::FatHead;

use Moo::Role;
use MooX::Options;

with qw( App::DuckPAN::Cmd );

option output_txt => (
    ...
    trigger => \&_initialize_fathead_output,
);

sub _initialize_fathead_output {
    my $self = shift;
    $self->app->fathead_output(path($self->output_txt));   
}
```

Then you could just do `with qw( App::DuckPAN::Option::FatHead )` whenever you need to use it.